sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -378,6 +378,8 @@ class AfterCount(TriggerFn):
   COUNT_TAG = _CombiningValueStateTag('count', combiners.CountCombineFn())
 
   def __init__(self, count):
```

Comment:
<!--new_thread; commit:17604fb4ecfba607daa3c22312c02865b02ecee7; resolved:0-->
Perhaps also have a JIRA issue, so that this can be fixed in other sdks.